docs/source/models/cause.rst

```diff
@@ -4,17 +4,119 @@
 Modeling Causes
 ===============
 
-.. todo::
-
-   Overview of modeling GBD causes.
+What is included in this chapter: models of causes that are represented as
+finite state Markov chain (this chapter will also explain what a finite state
+Markov chain is!)
+
+There are models of things that you might consider causes which are outside of
+the scope of this chapter, because they do not fit into the Markov approach.
+This chapter does not include “PAF of 1” causes, like protein-energy
+malnutrition (PEM).  It also does not include things that are sometimes
+considered a disease but are deemed “risk factors” by GBD, such as obesity.
+Unsafe water would not fit into the cause model paradigm either---it is a cause
+of diarrheal disease, but it is a risk factor in the GBD taxonomy.  Latent
+tuberculosis infection (LTBI) does fit into this chapter, but just barely.
 
 .. contents:
+   :local:
+
+
+What is a "cause"?
+------------------
+
+Short answer: disease
+
+More nuance: “cause of death” as might be included as the bottom line of the
+top half of a death certificate. And also analogous causes of nonfatal health
+loss, too.
+
+.. image:: death_certificate.png
+
+
+Learning objectives
+-------------------
+
+After reading this chapter, learners should be able to:
 
-What is a cause?
+1. Develop an understanding of how GBD, literature, and experts think about
+   a cause.
+2. Build internally consistent models which are sufficiently complex given
+   larger modeling goals.
+
+   a. Models that are as simple as possible, but no simpler.
+   b. Models that agree with withheld data.
+   c. Models that captures the outcomes of interest. (Which is really the same
+      as “but no simpler” in (a))
+
+3. Document the models in a way software engineers can build and validate it,
+   and document their understanding comprehensively for future researchers
+   (including their future selves) who are faced with related modeling
+   challenges.
+
+
+What is a cause model?
+----------------------
+
+A cause model is a simplification of a cause of death or disease for the
+purposes of simulation modeling.  It is always an idealization of the messy
+complexity of reality, and is designed to be acceptable to outside experts on
+the cause, as well as be parsimonious with the data available from the GBD that
+might inform the model.
+
+
+How is a cause model incorporated into a larger model?
+------------------------------------------------------
+
+Our modular structure is designed to layer cause models into the
+entity-component system that has a demographic model.  Sometimes an
```

Comment:
I'm uncertain how much a reference to a software pattern is helpful here.  

I also think splitting this cleanly into two ideas is helpful:
- How does a cause model affect other parts of the model.
- How is a cause model affected by other parts of the model.